File: JDK11/opentraceability/src/main/java/opentraceability/models/common/SBDHDocumentIdentification.java
package opentraceability.models.common;

import java.time.OffsetDateTime;
import opentraceability.utility.attributes.OpenTraceabilityAttribute;
import static opentraceability.Constants.SBDH_NAMESPACE;

public class SBDHDocumentIdentification {
    @OpenTraceabilityAttribute(ns = SBDH_NAMESPACE, name="Standard", sequenceOrder = 1)
    public String Standard = "";

    @OpenTraceabilityAttribute(ns = SBDH_NAMESPACE, name="TypeVersion", sequenceOrder = 2)
    public String TypeVersion = "";

    @OpenTraceabilityAttribute(ns = SBDH_NAMESPACE, name="InstanceIdentifier", sequenceOrder = 3)
    public String InstanceIdentifier = null;

    @OpenTraceabilityAttribute(ns = SBDH_NAMESPACE, name="Type", sequenceOrder = 4)
    public String Type = "";

    @OpenTraceabilityAttribute(ns = SBDH_NAMESPACE, name="MultipleType", sequenceOrder = 5)
    public String MultipleType = "";

    @OpenTraceabilityAttribute(ns = SBDH_NAMESPACE, name="CreationDateAndTime", sequenceOrder = 6)
    public OffsetDateTime CreationDateAndTime = null;

}

File: Java/OpenTraceability/src/main/opentraceability/models/common/sbdhdocumentidentification.kt
package models.common

import Constants
import utility.attributes.OpenTraceabilityAttribute
import java.lang.reflect.Type
import java.time.OffsetDateTime

class SBDHDocumentIdentification {

    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "Standard", 1)
    var Standard: String = ""

    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "TypeVersion", 2)
    var TypeVersion: String = ""

    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "InstanceIdentifier", 3)
    var InstanceIdentifier: String = ""


    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "Type", 4)
    var Type: String = ""

    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "MultipleType", 5)
    var MultipleType: String = ""


    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "CreationDateAndTime", 6)
    var CreationDateAndTime: OffsetDateTime? = null

}
